onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -3984,18 +3995,18 @@ def _aten_native_layer_norm_onnx(
 ) -> Tuple[TReal, TReal, TReal]:
```

Comment:
axes should be input now.

```python
axes: Sequence[INT64],
```

And other places using ReduceMax/ReduceMean/ReduceMin should all be updated, otherwise, the model would pop errors saying ReduceXXX having unexpected input/attribute axes depending on opset_verison 17/18.

Reply:
Not necessarily. The interface of an aten op/function shouldn't change because of its implementation. Since an attribute can be promoted to be an input, it should be okay to leave the interface as is. But it is important to ensure that the implementation works correctly. Eg., use the `noop_with_empty_axes` attribute as appropriate to ensure correct behavior for when axes is empty, etc.

Reply:
I am saying that the annotation should ideally depend on the aten specification (and not on the onnx opset). The call to onnx ops like ReduceMean, of course, will depend on the onnx opset.

Reply:
Make sense, but for onnx function (aten interface), I think we still rely on function_ir to differentiate attrs/inputs. I guess I am not sure if function_ir changed due to the opset version bump 17 -> 18 in this case?

Reply:
I think the idea might be related to https://github.com/microsoft/onnx-script/issues/443

Reply:
> I am saying that the annotation should ideally depend on the aten specification (and not on the onnx opset). The call to onnx ops like ReduceMean, of course, will depend on the onnx opset.

I agree. We _could_ change it to `INT64` (instead of `Sequence[INT64]`) prepare for symint inputs. The evaluator should be able to handle this (maybe already)